sdks/python/apache_beam/runners/interactive/caching/streaming_cache.py

```diff
@@ -315,6 +327,7 @@ def read_multiple(self, labels):
         StreamingCacheSource(self._cache_dir, l,
                              self._is_cache_complete).read(tail=True)
         for l in labels
+        if not [sub_l for sub_l in l if self.sentinel_label() in sub_l]
```

Comment:
This is a little hard to read. 
Isn't a label `l` a `str`, so a `sub_l` is a character of that `str`?
I suppose `if not [sub_l for ...]` evaluates to `True` when the `[sub_l for ...]` is empty.
And the emptiness of `[sub_l for ...]` is based on whether the `sentinel_label` exists in the `sub_l`? This is where I get confused.

Reply:
Or if a label `l` is a `list` of `str` and `*labels` is a `list` of `list` of `str`, then this makes sense.

Reply:
Sorry, I changed the PR and it looks like your comment is out of date. Can you PTAL?